include/EVT/dev/RTCTimer.hpp

```diff
@@ -0,0 +1,96 @@
+#ifndef EVT_TIMER_H
+#define EVT_TIMER_H
+
+#include <EVT/dev/RTC.hpp>
+#include <EVT/dev/Timer.hpp>
+#include <EVT/utils/time.hpp>
+
+namespace time = EVT::core::time;
+
+namespace EVT::core::DEV {
+
+class RTCTimer : public Timer {
+public:
+    /**
+     * Create instance of RTCTimer.
+     * clock period defaults to 1s.
+     *
+     * @param rtc
+     */
+    RTCTimer(RTC& r);
+
+    /**
+     * Create instance of RTCTimer.
+     *
+     * @param rtc Instance of on-board
+     * @param clockPeriod Amount of time it takes for the time to go off in ms
```

Comment:
NITPICK: The parameter name is `clock` in the code and `clockPeriod` in the docstring